docarray/array/mixins/evaluation.py

```diff
@@ -48,7 +48,7 @@ def evaluate(
         hash_fn: Optional[Callable[['Document'], str]] = None,
         metric_name: Optional[str] = None,
         strict: bool = True,
-        label_tag: Optional[str] = 'label',
+        label_tag: str = 'label',
```

Comment:
I'd rather check what happens with `None`. One should be able to put `None`

Reply:
Ok so do you mean, that it should be optional and default to `label`.  If a user does not want to use labels, i.e., pass a `ground_truth` it should be ok to pass `None`. However, if it is set to `None` and no `ground_truth` is passed, an exception is raised.